frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
I tried to merge this into master, but had to revert again, because it breaks the loading of issues. For some reason the app starts reloading all issues on every click on an issue on the map.